pkgs/by-name/op/ophcrack-cli/package.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, fetchpatch
+, autoreconfHook
+, libtool
+, pkg-config
+, zlib
+, openssl
+, freetype
+, gcc
+, expat
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ophcrack";
+  version = "3.8.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/ophcrack/files/ophcrack/3.8.0/ophcrack-3.8.0.tar.bz2";
+    hash = "sha256-BIpt9XmDo6WjGsfE7BLfFqpJ5lKilnbZPU75WdUK7uA=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    libtool
+  ];
+
+  buildInputs = [
+    pkg-config
+    zlib
+    openssl
+    freetype
+    gcc
+    expat
+  ];
+
+  patches = [
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/c60118b40802e1162dcebfe5f881cf973b2334d3/debian/patches/fix_spelling_error.diff";
+      hash = "sha256-Fc044hTU4Mtdym+HukGAwGzaLm7aVzV9KpvHvFUG2Sc=";
+    })
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/e19d993a7dbf131d13128366e2aac270a685befc/debian/patches/qmake_crossbuild.diff";
+      hash = "sha256-sOKXOBpAYGLacU6IxjRzy3HCnGm4DFowDL2qP+DzG8M=";
+    })
+  ];
+
+  dontWrapQtApps = true;
```

Comment:
@Tochiaha I assume this is not needed anymore ?

Reply:
yeah it isn't needed anymore.